docs/conf.py

```diff
@@ -271,6 +272,7 @@
 html_use_index = True
 
 # html_css_files = ["custom.css", ("print.css", {"media": "print"})]
```

Comment:
Once you have an actual usage, it's a good idea to remove now obsolete usage comments. It helps keep your code clean and maintainable.

```suggestion
```